docs/running-on-kubernetes.md

```diff
@@ -477,6 +477,22 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
+  <td><code>spark.kubernetes.executor.labels</code></td>
+  <td>(none)</td>
+  <td>
+    Custom labels that will be added to the executor pods. This should be a comma-separated list of label key-value pairs,
+    where each label is in the format <code>key=value</code>.
```

Comment:
how do you put a label where the value contains an `=`?  Or a comma `,` ?

`key1=value1,key2=value2=with-equals,key3=value3` ?